src/teleop/gui/src/components/AutonRoverMap.vue

```diff
@@ -0,0 +1,349 @@
+    //   })
+
+    //   this.projectedPointsType = msg.path_type
+    // })
+
+    // this.$parent.subscribe('/estimated_gate_location', (msg) => {
+    //   this.post1 =  L.latLng(
+    //     msg.post1.latitude_deg + msg.post1.latitude_min/60,
+    //     msg.post1.longitude_deg + msg.post1.longitude_min/60
+    //   )
+    //   this.post2 = L.latLng(
+    //     msg.post2.latitude_deg + msg.post2.latitude_min/60,
+    //     msg.post2.longitude_deg + msg.post2.longitude_min/60
+    //   )
+    // })
+
+  },
+
+  computed: {
+    ...mapGetters('autonomy', {
+      route: 'route',
+      waypointList: 'waypointList',
+    }),
+
+    // Convert to latLng object for Leaflet to use
+    odomLatLng: function () {
+      return L.latLng(this.odom.latitude_deg + this.odom.latitude_min/60, this.odom.longitude_deg + this.odom.longitude_min/60)
+    },
+    
+    // Concat waypoints on course with rover marker at index 0 for polyline
+    polylinePath: function () {
+      return [this.odomLatLng].concat(this.route.map(waypoint => waypoint.latLng))
+    },
+    
+    // Concat waypoints on projected course with rover marker at index 0 for polyline
+    projectedPath: function () {
+      return [this.odomLatLng].concat(this.projectedPoints.map(projected_point => projected_point.latLng))
+    },
+
+  },
+
+  data () {
+    return {
+      center: L.latLng(38.406371, -110.791954),
+      attribution: '&copy; <a href="http://osm.org/copyright">OpenStreetMap</a> contributors',
+      online: true,
+      onlineUrl: onlineUrl,
+      offlineUrl: offlineUrl,
+      roverMarker: null,
+      waypointIcon: null,
+      map: null,
+      odomCount: 0,
+      locationIcon: null,
+      odomPath: [],
+
+      tangentMarker: null,
+      tangentIcon: null,
+
+      targetBearingMarker: null,
+      targetBearingIcon: null,
+
+      projectedPoints: [],
+      projectedPointsType: '',
+
+      post1: null,
+      post2: null,
+
+      findRover: false,
+
+      options: {
+        type: Object,
+        default: () => ({})
+      },
+      tileLayerOptions: {
+        maxNativeZoom: 18,
+        maxZoom: 100
+      }
+    }
+  },
+
+  props: {
+    odom: {
+      type: Object,
+      required: true
+    },
+    GPS: {
+      type: Object,
+      required: true
+    },
+    TargetBearing: {
+      type: Object,
+      required: true
+    }
+  },
+
+  methods: {
+    // Event listener for setting store values to get data to waypoint Editor
+    getClickedLatLon: function (e) {
+      this.setClickPoint(
+          { 
+            lat: e.latlng.lat,
+            lon: e.latlng.lng
+          }
+        )
+    },
+
+    ...mapMutations('autonomy',{
+      setClickPoint: 'setClickPoint',
+      setWaypointList: 'setWaypointList',
+      setOdomFormat: 'setOdomFormat'
+    }),
+  },
+
+
+  watch: {
+    odom: {
+      handler: function (val) {
+        // Trigger every time rover odom is changed
+  
+        const lat = val.latitude_deg + val.latitude_min / 60
```

Comment:
I believe we were talking at the end of last year about how there's really no point to ever not just lat/lng completely in decimal format, without using minutes and seconds (except for inputting, since the competition previously required us to be able to input in minutes/seconds)

Reply:
Everything internal, after the input, should be a double lat value and a double lng value

Reply:
Will add a commit for this later today